src/server/routes/nyplApiClient/index.js

```diff
@@ -1,27 +1,35 @@
 import NyplApiClient from '@nypl/nypl-data-api-client';
-import config from '../../../../appConfig.js';
 import aws from 'aws-sdk';
 
+import config from '../../../../appConfig.js';
+import logger from '../../../../logger.js';
+
 const appEnvironment = process.env.APP_ENV || 'production';
+const kmsEnvironment = process.env.KMS_ENV || 'encrypted';
 const apiBase = config.api[appEnvironment];
-const kms = new aws.KMS({
-  region: 'us-east-1',
-});
+let decryptKMS;
+let kms;
 
-function decryptKMS(key) {
-  const params = {
-    CiphertextBlob: new Buffer(key, 'base64'),
-  };
+if (kmsEnvironment === 'encrypted') {
```

Comment:
I feel it will be much more easier to help me understand. Not saying the current version is not a descent practice though!

The other thing I am thinking is, if there's no `decryptKMS` in line 62, shouldn't we just not run the Promise function at all?  `catch` to me is for unexpected error, but in this case, we know sometimes we won't have `decryptKMS`.